examples/basic-cdk/cdk.json

```diff
@@ -0,0 +1,34 @@
+{
+  "app": "npx ts-node --prefer-ts-exts bin/stack.ts",
```

Comment:
Let's add ts-node to nearest package dependencies / devDependencies so we are sure which version is used - this has blocked us from upgrading ts in the past, as it used some old ts-node version from nested package

Reply:
done